source/main/index.ts

```diff
@@ -256,7 +263,57 @@ const onAppReady = async () => {
     client.create(mainWindow);
   }
 
+  // Register custom browser protocol
+
+  // TODO: this is unethical! user hostility – don’t touch my default programs, esp. when not asking me – @michalrus
```

Comment:
I think this shouldn’t happen like that.

Right now, we re-associate `web+cardano://` to Daedalus on Windows and macOS on each application start, disregarding what the user might want.

Linux is set up correctly – associating `web+cardano://` once, in the installer.

I think we should do the same thing for Windows and macOS – associate *once* during installation, and never re-associate.

Reply:
/cc @angerman – a bit similar to auto-adding the app to macOS dock, which you found hostile – https://input-output-rnd.slack.com/archives/GJMEW8FV3/p1647944259851109?thread_ts=1647861796.476749&cid=GJMEW8FV3